app.js

```diff
@@ -19,11 +19,19 @@
 
 const fs = require("fs");
 const { jsonToSchema } = require("./lib");
+const yargs = require("yargs/yargs");
+const { hideBin } = require("yargs/helpers");
+const argv = yargs(hideBin(process.argv)).argv;
 
 const main = () => {
   const jsonInput = fs.readFileSync(0, "utf8");
-
-  const { error, value } = jsonToSchema({ jsonInput });
+  const baseType = argv.baseType || "AutogeneratedMainType";
+  const prefix = argv.prefix || "";
```

Comment:
If you use the defaults, these lines can be changed to:
```suggestion
  const { baseType, prefix } = argv;
```

Reply:
Ok, done. Sorry but I'm not a nodejs/javascript developer. Thanks